### PR TITLE
2083 country drop down

### PR DESCRIPTION
**Pull request by sergioloporto**

### CODE REVIEW

File: verification/curator-service/ui/src/redux/store.ts [hunk 1]
```diff
@@ -2,17 +2,20 @@ import { configureStore, combineReducers } from '@reduxjs/toolkit';
 
 import appReducer from './app/slice';
 import authReducer from './auth/slice';
+import countryListReducer from './filters/slice';
```
Comment (maciekz1996) [inline verification/curator-service/ui/src/redux/store.ts:5]: To make things consistent you can change the name here from `countryListReducer` to `filtersReducer`

File: verification/curator-service/ui/src/redux/filters/thunk.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const countryListJsonFile =
+    'https://covid-19-aggregates.s3.amazonaws.com/country/latest.json';
+
+type countryObject = {
+    _id: string
+}
+
+export const fetchCountries = createAsyncThunk(
+    'countryList/fetchCountries',
```
Comment (maciekz1996) [inline verification/curator-service/ui/src/redux/filters/thunk.ts:12]: Here you can change `countryList/fetchCountries` to `filters/fetchCountries` to make things consistent

File: verification/curator-service/ui/src/redux/filters/slice.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    isLoading: false
+};
+
+interface initialStateTypes {
+    countryList:string[],
+    isLoading: boolean
+  }
+
+const slice = createSlice({
+    name: 'countryList',
```
Comment (maciekz1996) [inline verification/curator-service/ui/src/redux/filters/slice.ts:15]: You can change the name to `filters`

File: verification/curator-service/ui/src/redux/@types/react-redux.d.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import 'react-redux';
```
Comment (maciekz1996) [inline verification/curator-service/ui/src/redux/@types/react-redux.d.ts:1]: What was the purpose of adding those custom types?

**Review by maciekz1996 — CHANGES_REQUESTED** [review]

File: verification/curator-service/ui/src/redux/@types/react-redux.d.ts (hunk 4 above)
Comment (sergioloporto) [reply verification/curator-service/ui/src/redux/@types/react-redux.d.ts:1]: I was getting ts errors, and I found that it is a good practice to add that custom type

**Review by maciekz1996 — CHANGES_REQUESTED** [review]

File: verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts [hunk 5]
```diff
@@ -454,7 +468,7 @@ describe('Linelist table', function () {
         cy.contains('7 rows selected').should('not.exist');
     });
 
-    it('Can delete all cases across rows for a search result', function () {
+    it.only('Can delete all cases across rows for a search result', function () {
```
Comment (maciekz1996) [inline verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts:471]: This `only` is preventing other tests from running in this test file

**Review by maciekz1996 — CHANGES_REQUESTED** [review]

**Review by maciekz1996 — APPROVED** [review]

**Review by iamleeg — APPROVED** [review]
Looks great, nice one. Some slight data concerns (Gibraltar is listed as a country, officially it isn't, for example; Haiti and Honduras are missing) but we already have a couple of different places where Mapbox and other sources don't agree on this. I think there's no value and a lot of time spent in trying to build a "canonical" list, so let's leave it as it is and if it turns out that someone can't do something they want we'll address it then.

**Comment by sergioloporto** [comment]
@iamleeg agreed, thank you

**Pull request merged**